docs/pages/docs/reference/command-line-reference.mdx

```diff
@@ -454,6 +454,16 @@ Defaults to `https://vercel.com`.
 
 Defaults to `https://vercel.com/api`.
 
+#### `--sso-team`
+
+`type: string`
+
+Connect to an sso-enabled Vercel team by providing your Team slug.
```

Comment:
Does this deserve a callout that it will also `link` your repo by writing the `.turbo/config.json`?

Reply:
Can we update the `turbo` to state this when it happens? So it becomes self-evident?